lib/jwt/jwt.go

```diff
@@ -306,35 +306,42 @@ type SignParamsJWTSVID struct {
 func (k *Key) SignJWTSVID(p SignParamsJWTSVID) (string, error) {
 	// Record time here for consistency between exp and iat.
 	now := k.config.Clock.Now()
-	claims := jwt.Claims{
+
+	claims := map[string]any{
```

Comment:
I would suggest adding a comment: 



```suggestion
   // We use map[string]any instead of jwt.Claims to avoid a json.Marshal/Unmarshal  
  // round-trip that would convert jwt.NumericDate (int64) to float64, causing      
  // timestamp claims to be serialized in scientific notation (e.g., "exp": 1.7e9). 
  // Using map[string]any preserves the jwt.NumericDate type until final marshaling.
	claims := map[string]any{
```